libraries/fault_sdk/fsInjection.c

```diff
@@ -7,20 +7,14 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <fm_util.h>
+#include <inttypes.h>
 #include "fsInjection.h"
 
-float fsInjection(int cmdToFaultSensor)
-{
-	/*Definition of Variables*/
-	float theta_hat_rel,theta_rel,delta_theta_rel;	
-	
-	if (cmdToFaultSensor == 1)
-	{
-		delta_theta_rel = 10; /*TODO change to better bias*/
-	}
-	else
-	{
-		delta_theta_rel = 0; /*No bias if not commanded to fault*/;
+
+void fsInjection(double *deltaThetaRadFine1, FmState *fmState)
```

Comment:
Let's change `deltaThetaRadFine1` to`deltaTheta`. The naming here (in fsInjection) should be agnostic of the system in general.